integrations/email/email-incidents/README.md

```diff
@@ -0,0 +1,84 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The Email Incidents integration provides a pipeline for sending emails about incidents from Sensu.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `email` [pipeline]
+* `email-handler` [handler]
+* `smtp_user` [secret]
+* `smtp_password` [secret]
+* `sensu/sensu-email-handler:1.2.2` [asset]
+
+## Dashboards
+
+<!-- List of compatible dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+There are no compatible dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Configure Secrets Management
+
+   This integration requires the following [Sensu Secrets][secrets]:
+
+   - `smtp_user`
+   - `smtp_password`
+
+   _NOTE: this integration creates one or more Sensu Secrets using the "env" provider by default. When using the "env" provider, the corresponding environment variables need to be set on every sensu-backend in the Sensu deployment. To add the environment variables, please modify `/etc/default/sensu-backend` or `/etc/sysconfig/sensu-backend` and restart the sensu-backend service(s)._
```

Comment:
```suggestion
```

I think we can leave this out since you have added support for configuring secrets via prompts.